examples/vite/package.json

```diff
@@ -3,11 +3,12 @@
     "dev": "npm run server",
     "prod": "npm run build && npm run server:prod",
     "build": "vite build && vite build --ssr",
-    "server": "node server.js",
+    "server": "esno ./server",
```

Comment:
We don't need `esno` since the server is still JS.

Reply:
True!